addons/udes_stock/tests/test_stock_inventory.py

```diff
@@ -0,0 +1,87 @@
+from odoo.exceptions import ValidationError
+from . import common
+
+class TestStockInventory(common.BaseUDES):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestStockInventory, cls).setUpClass()
+        cls.apple_quant = cls.create_quant(cls.apple.id, cls.test_location_01.id, 5)
+
+    def setUp(self):
```

Comment:
please could you add doc strings to all methods

Reply:
Added